R-package/R/plot_results.r

```diff
@@ -12,7 +12,7 @@
 #' 3) a raster map with delineated habitat types -> HabitatMap_final.tif
 #'
 #' @export
-plot_Results <- function(inPath, threshold, color = NULL) {
+plot_Results <- function(inPath, color = NULL) {
```

Comment:
thanks! and we remove "@param threshold probability threshold provided by the user."

Reply:
Yes, and documentation updated with: 63ebe8d